### ISSUE TITLE
kernel-debug-debuginfo files not detected as "debugpkg" (anymore?)

### ISSUE THREAD

**Issue by jankaluza** [rpminspect/rpminspect#1314]
**Describe the bug**
This is related to OSCI-6002. When executing rpminspect 2.0 for kernel, it reports following issues:

```
/usr/lib/debug/lib/modules/5.14.0-362.15.1.el9_3.aarch64+debug/vdso/vdso.so.debug in kernel-debug-debuginfo-5.14.0-362.15.1.el9_3 on aarch64 contains debugging symbols.
Contains: .symtab
```

Checking the code briefly, it seems that this message is printed only if rpminspect detects that this file is *not* a debuginfo package.

**To Reproduce**
```
/usr/bin/rpminspect -c /usr/share/rpminspect/redhat.yaml --workdir /tmp --format=json --output=./result.json --verbose --release=el9 kernel-5.14.0-362.14.1.el9_3 kernel-5.14.0-362.15.1.el9_3
```

**Expected behavior**
The error message is not printed, because it is expected for debuginfo files to have the `.symtab` symbols.

**Environment:**
- RHEL-9.3.0

**Comment by dcantrell** [comment]
I have been unable to reproduce this locally.  Are you still seeing this?

**Comment by jankaluza** [comment]
Yes, we still see it, for example here: http://artifacts.osci.redhat.com/testing-farm/051f99c5-c2d3-4fa1-b805-6b3e77aacb9d/work-rpminspectyjb39jrt/rpminspect/execute/data/guest/default-0/rpminspect-1/data/viewer.html.

You can see the rpminspect version used in the "Diagnostics".

**Comment by dcantrell** [comment]
OK, the latest build should have this problem resolved.  There are two things that are showing up for me in the debuginfo inspection as BAD results:

1. ```/usr/libexec/kselftests/bpf/*``` files are reported as containing .symtab.  rpminspect is expecting these files to be stripped because they are not in a debuginfo package.  However, the bpf stuff may be different in which case the kernel package needs to add this to its rpminspect.yaml file:

```
debuginfo:
    ignore:
        - /usr/libexec/kselftests/bpf/*
```

2. ```/usr/lib/debug/usr/libexec/perf-core/tests/shell/coresight/asm_pure_loop/asm_pure_loop.debug``` and ```/usr/lib/debug/usr/libexec/perf-core/tests/shell/coresight/memcpy_thread/memcpy_thread.debug``` report as missing .gdb_index.  Should they have that?  I don't know.  **rpminspect-data-redhat** is configured to look for ```.symtab```, ```.debug_info```, and ```.gdb_index``` in debuginfo files.  In Fedora we only look for ```.symtab``` and ```.debug_info```.  Should rpminspect-data-redhat drop .gdb_index?
